src/config/userTypes.ts

```diff
@@ -32,33 +32,43 @@ export interface IHacker {
   id: string;
   accountId: string | IAccount; // for querying account as well
   status: HackerStatus;
-  school: string;
-  degree: string;
-  // no enum for these
-  gender?: string;
-  needsBus?: boolean;
   application: {
-    portfolioURL: {
-      // gcloud bucket link
-      resume: string;
-      github?: string;
-      dropler?: string;
-      personal?: string;
-      linkedIn?: string;
-      other?: string;
+    general: {
+      school: string;
+      degree: string;
+      fieldOfStudy: string;
+      graduationYear: number;
+      jobInterest: JobInterest;
+      URL: {
+        resume: string;
+        github?: string;
+        dribbble?: string;
+        personal?: string;
+        linkedIn?: string;
+        other?: string;
+      };
+    };
+    shortAnswer: {
+      // array of mongoose ids referencing different skills
+      skills?: string[];
+      // any miscelaneous comments that the user has
+      comments?: string;
+      // "Why do you want to come to our hackathon?"
+      question1: string;
+      // "Some q"
+      question2: string;
+    };
+    other: {
+      // no enum for these
+      ethnicity: string[];
+      gender: string;
```

Comment:
`gender` is moved to profile now

Reply:
There is no section for profile, should I create it? Or should it be part of the Account userType instead?

Reply:
profile = account creation, @logan-r added it in his pr